packages/loot-core/src/shared/util.ts

```
@@ -353,65 +368,69 @@
         .replace(getNumberFormat().regex, '')
         .replace(getNumberFormat().separatorRegex, '.'),
     );
   } else {
     amount = parseFloat(
-      str
+      currencyAmount
         .replace(getNumberFormat().regex, '')
         .replace(getNumberFormat().separator, '.'),
     );
   }
   return isNaN(amount) ? null : amount;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)